internal/cli/cmd/cluster/docker.go

```diff
@@ -126,15 +129,158 @@ func NewDockerLoginCmd(hidden bool) *cobra.Command {
 			return fnerrors.New("failed to write %q: %w", cfgFile, err)
 		}
 
-		if nscr := response.NSCR; nscr != nil {
-			fmt.Fprintf(stdout, "\nYou are now logged into your Workspace container registry:\n\n  %s/%s", nscr.EndpointAddress, nscr.Repository)
-			fmt.Fprintf(stdout, "\n\nRun your first build with:\n\n  $ nsc build . -n test:v0.0.1 -p")
+		if _, err := exec.LookPath(credHelperBinary); err != nil {
+			if errors.Is(err, exec.ErrNotFound) {
+				fmt.Fprintf(stdout, "\nWarning: nsc binary is not installed in your $PATH. Docker credentials won't work until you add nsc's path to the $PATH.\n")
```

Comment:
If we modify the configuration, then I suggest always printing the block below, with next steps.

And adding a warning at the end (as that's last thing that people read). And I would suggest something more like:

"We didn't find docker-credentials-nsc in your $PATH. It's usually installed along-side nsc; so if you have nsc to the $PATH, docker-credentials-nsc will also be available.

While your $PATH is not updated, accessing nscr.io images from docker-based tools won't work. But you can always use `nsc build` (as per above) or `nsc run`."